airbyte-webapp/src/components/ui/Popout/Popout.tsx

```diff
@@ -83,7 +75,17 @@ export const Popout: React.FC<PopoutProps> = ({ onChange, targetComponent, ...pr
         onChange={onSelectChange}
         components={components}
       />
-      {isOpen ? <OutsideClickListener onClick={toggleOpen} /> : null}
+      {isOpen && (
+        <div
```

Comment:
does it make sense to use some variation of the `Overlay` component here instead?  I am not sure that it makes enough difference, but we could add a `transparent` prop to it that removes the background shading.

Reply:
Yep that makes sense, I will make the update